recipes/syngap/meta.yaml

```diff
@@ -93,6 +81,7 @@ test:
     - syngap genepair -h
     - syngap evi -h
     - syngap eviplot -h
+    - df -h
```

Comment:
ditto

Reply:
> ditto

It's so wired that when I followed your suggestions, the the Azure CI met the error like below https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=61168&view=logs&s=5162d36d-59e0-5d46-757c-443386a65b88&j=e14e69ff-a0ae-55c4-b71d-229b239cfb2f
![image](https://github.com/bioconda/bioconda-recipes/assets/90335707/c5110c9a-a85c-4816-870a-cec8f886b7a8)
![image](https://github.com/bioconda/bioconda-recipes/assets/90335707/a77b4a1e-2ae8-4b98-b9b0-4e0937a39765)

Reply:
And here is the previous successful run before the modification, which didn't meet the "no space left on device" error.
https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=61117&view=logs&j=e14e69ff-a0ae-55c4-b71d-229b239cfb2f&t=4dddc55b-10bc-50fc-ac68-7ff899b7a030